torch_ort/experimental/debug.py

```diff
@@ -0,0 +1,73 @@
+import os
+from torch_ort import ORTModule
+from onnxruntime.training.ortmodule._logger import LogLevel
+
+
+def save_intermediate_onnx_graphs(model: ORTModule, prefix: str) -> bool:
```

Comment:
Do you also think we should expose a way to toggle saving the onnx graphs with a toggle flag? Also what is the intention of returning a ```bool```?

Reply:
Good point. I was in doubt whether we should expose the toggle. I've tried to be very minimalistic, but I will add the flag